smartfile/__init__.py

```diff
@@ -230,30 +272,33 @@ def get_request_token(self, callback=None):
                            client_secret=self._client.secret,
                            callback_uri=callback,
                            signature_method=SIGNATURE_PLAINTEXT)
-            r = requests.post(urlparse.urljoin(self.url, 'oauth/request_token/'), auth=oauth)
+            r = requests.post(urlparse.urljoin(
+                self.url, 'oauth/request_token/'), auth=oauth)
             credentials = urlparse.parse_qs(r.text)
             self.__request = OAuthToken(credentials.get('oauth_token')[0],
-                                        credentials.get('oauth_token_secret')[0])
+                                        credentials.get(
+                                        'oauth_token_secret')[0])
             return self.__request
 
         def get_authorization_url(self, request=None):
             "The second step of the OAuth workflow."
             if request is None:
                 if not self.__request.is_valid():
-                    raise APIError('You must obtain a request token to request '
-                                   'and access token. Use get_request_token() '
-                                   'first.')
+                    raise APIError('You must obtain a request token to'
+                                   'request and access token. Use'
+                                   'get_request_token() first.')
                 request = self.__request
             url = urlparse.urljoin(self.url, 'oauth/authorize/')
-            return url + '?' + urllib.urlencode(dict(oauth_token=request.token))
+            return url + '?' + urllib.urlencode(
+                dict(oauth_token=request.token))
 
         def get_access_token(self, request=None, verifier=None):
-            """The final step of the OAuth workflow. After this the client can make
-            API calls."""
+            """The final step of the OAuth workflow. After this the client
+            can make API calls."""
             if request is None:
                 if not self.__request.is_valid():
-                    raise APIError('You must obtain a request token to request '
-                                   'and access token. Use get_request_token() '
+                    raise APIError('You must obtain a request token to request'
+                                   'and access token. Use get_request_token()'
```

Comment:
You actually do need those spaces before the ends of the string -- otherwise it will render like this: "You must obtain a request token to requestand access token" Also, "and" should be "an". :palm_tree: